test/fixtures/catalog/when_condition_check.yaml

```diff
@@ -0,0 +1,33 @@
+id: when_condition_check
```

Comment:
Do we need a whole new fixture to test this new field?
Couldn't we use an already existing one and add the field there?

Reply:
+ 1, i think we can squash several checks fixtures in just two of them, one being the "valid" and one being "malformed"